src/components/common/base-card/base-card.tsx

```diff
@@ -0,0 +1,15 @@
+import { Body } from "./base-card.styles";
+
+interface IProps {
+    children: React.ReactNode;
+    focused?: boolean;
+    onClick?: (event: React.MouseEvent<HTMLDivElement>) => void;
+}
+
+export const BaseCard = ({ children, focused, onClick }: IProps) => {
+    return (
+        <Body tabIndex={0} $focused={focused} onClick={onClick}>
```

Comment:
rename as `CardBody` to be clearer

Reply:
But it is within the BaseCard though. I thought it would be quite clear

Reply:
I actually mistook it as the `<body>` initially but if it does not look that misleading, I am ok to close it

Reply:
I am referencing the style file so i think it is fine. It's an internal component anyway